src/main/java/gui/RootPane.java

```diff
@@ -0,0 +1,50 @@
+package gui;
+
+import javafx.application.Application;
+import javafx.scene.Scene;
+import javafx.scene.layout.BorderPane;
+import javafx.stage.Stage;
+
+/**
+ * Class that represents the whole top of the gui application.
+ */
+public class RootPane extends Application {
+
+    private int minResX = 640;
+    private int minResY = 480;
+    private int startResX = 800;
+    private int startResY = 600;
+
+    //CHECKSTYLE:OFF: MagicNumber
+    @Override
+    public void start(Stage primaryStage) throws Exception {
+
+        // Create a BorderPane,
+        // a layout with 5 areas: top, bottom, left, right and center,
+        // and add our views to it.
+
+        BorderPane topLevelPane = new BorderPane();
```

Comment:
Could you make all the panes fields of RootPane. This gives better overview of the parts of RootPane